### DISCUSSION TITLE
Schematron rules on GovernmentAgreementConstraintIndicator in combination with PartyTypeCode

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#840] [Q&A] [answered]
Hi, i saw that in 1.10 there were rules added about the partytypecode in combination with the GovernmentAgreementConstraintIndicator. As in the example below. First i dont understand the reason for this schematrons. I also saw that these rules will be removed in SDK1.11.  Is that correct that they will be removed? And what is the reason that they were added and will be removed one SDK version later?

	<rule context="/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingProcess[$noticeSubType = '38']">
		<assert id="BR-BT-00115-0098" role="ERROR" diagnostics="BT-115-Lot" test="count(cbc:GovernmentAgreementConstraintIndicator) = 0 or (../../cac:ContractingParty/cac:ContractingPartyType/cbc:PartyTypeCode[@listName='buyer-legal-type']/normalize-space(text()) = ('cga','ra','la','body-pl','body-pl-cga','body-pl-ra','body-pl-la','pub-undert','pub-undert-cga','pub-undert-ra','pub-undert-la','org-sub-cga','org-sub-ra','org-sub-la','def-cont','int-org','eu-ins-bod-ag'))">rule|text|BR-BT-00115-0098</assert>
	</rule>

**Comment by YvesJo** [comment, marked as answer]
Hi,
Most of the rules were identified and stated during the Member States consultation. They were also ready early in the development process. As they added additional challenges, it was decided to have them removed from the early SDK and introduced progressively. 
The activation of that specific rule occurred with SDK 1.10. We are validating the rules based on some case scenarios, however we aren't able to cover all the cases; it may happen that under some specific conditions a rule is problematic and requires a corrective action. 
Depending on the priorities and the first analysis, the corrective action may result in a rule update or removal. Removal was the best option for that specific rule which will also get removed from SDK 1.10.
KR

**Reply by kletersteegp** [reply]
Ok thanks for your reply

**Reply by kletersteegp** [reply]
Can you give an indication when this will be backported to SDK1.10?

**Reply by YvesJo** [reply]
Hi,
Please get in touch with the helpdesk regarding release dates: ted@publications.europa.eu
